src/Tasks/Microsoft.NET.Build.Tasks/ResourceUpdater.cs

```diff
@@ -0,0 +1,320 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.NET.Build.Tasks
+{
+    /// <summary>
+    /// Provides methods for modifying the embedded native resources
+    /// in a PE image. It currently only works on Windows, because it
+    /// requires various kernel32 APIs.
+    /// </summary>
+    public class ResourceUpdater
```

Comment:
The only classes that should be public in this assembly are the build tasks.

I missed in prior reviews that the AppHost class was made public (though you changed it to internal, which is good.)

Why is this being made public?

Reply:
Just something I missed. I made it internal, thanks.